Cargo.toml

```diff
@@ -12,13 +12,14 @@ license = "MIT"
 exclude = ["media"]
 
 [features]
-default = ["native-tls", "ctrlc_handler", "teloxide-core/default",  "auto-send"]
+default = ["native-tls", "ctrlc_handler", "teloxide-core/default", "auto-send"]
```

Comment:
This extra space was erroneous, this is a good change (see my previous comment on formatting).